docs/neopixel.rst

```diff
@@ -0,0 +1,69 @@
+Neopixel
+****
+
+.. py:module:: neopixel
+
+The ``neopixel`` module lets you use Neopixel (WS2812) individually addressable RGB LED strips with the Microbit.    
+Note to use the ``neopixel`` module, you need to import it separately with::
+
+    import neopixel
+
+.. warning::
+    
+    Do not use the 3v connector on the Microbit to power any more than 8 Neopixels at a time.
```

Comment:
Current recommendation is max of 100mA from the edge connector
![image](https://cloud.githubusercontent.com/assets/83421/12690597/9a68615c-c6dc-11e5-93c4-9e6b08a1b0b5.png) (source: micro:bit student guide)